app/models.py

```diff
@@ -58,4 +56,28 @@ class OrderProduct(Base):
     created_at = Column(DateTime(timezone=True), default=lambda: datetime.now(timezone.utc))
     updated_at = Column(DateTime(timezone=True), onupdate=datetime.now(timezone.utc))
 
-    order = relationship("Order", back_populates="products")
\ No newline at end of file
+    order = relationship("Order", back_populates="products")
+
+class Product(Base):
+    __tablename__ = "products"
+    id = Column(PostgresUUID(as_uuid=True), primary_key=True, default=uuid.uuid4, unique=True, nullable=False)
```

Comment:
what is PostgresUUID? I can't see any import statement for it?

Reply:
Done